app/Http/Controllers/v1/UserController.php

```diff
@@ -11,6 +11,7 @@
 use Illuminate\Http\Response;
 use Illuminate\Support\Facades\Log;
 use Psr\Log\InvalidArgumentException;
+use Illuminate\Support\Facades\Cache;
```

Comment:
Imported but unused. Doesn't matter for this PR, but check if still needed by the next/final one.